docs/events/glides/com.mbta.ctd.glides.trips_updated.v1.mdx

```diff
@@ -0,0 +1,288 @@
+
+### Car
+
+Fields:
+
+- `label` (string | `"none"`, optional): car number corresponding to the GTFS-RT `label` field. Or `"none"` if the inspector has unassigned the car. If the field is absent, it is not modified.
+- `operator` ([Operator](.#operator) | `"none"` | `"unset"`, optional): the operator assigned to this car. The string `"none"` means the inspector has unassigned the operator without reassigning another. The string `"unset"` means to discard the previous value as if the field had never been modified. If the field is absent, it is not modified from its previous value.
+
+`"unset"` and `"none"` are semantically different, and used consistently with the `"unset"` values in [TripUpdated](#tripupdated). If an operator is `"unset"`, then, as if no update had ever been made, clients SHOULD assume the scheduled operator will operate the car. If an operator is `"none"`, then we don't know who will operate the car.
+
+An empty object `{}` is valid if nothing about the car has been modified.
+
+The length of `cars` always reflects the known length of the train.
+
+### DroppedReason
+
+An object representing the reason that a trip was dropped. The presence of this object implies that the trip was dropped.
+
+Fields:
+
+- `reason` (string): free-text description about why a trip was dropped.
+
+Currently the reason is entered by inspectors in a free-text field. It's published in an object so that if Glides collects more structured data in the future, then extra fields can be added. If more fields are added, then a generated text description would be filled into the `reason` field, and it would not be a breaking change.
+
+`reason` MAY be the empty string if for some reason Glides does not have information about why a trip was dropped.
+
+### Location
+
+See [Location](.#location)
+
+### Metadata
+
+See [Metadata](.#metadata)
+
+### Operator
+
+See [Operator](.#operator)
+
+### Scheduled
+
+Scheduled information about the trip. It was not updated in Glides, but is included in the event stream so that consumers can know the schedule information that Glides uses. If data here was never overriden by a corresponding field in [TripUpdated](#tripupdated), then consumers can assume that the trip operated based on the scheduled information contained here.
+
+Fields:
+
+- `scheduledCars` (array of [ScheduledCar](#scheduledcar)): Array of length 1 or 2. The length reflects the scheduled length of the train. In a 2 car train, the front car is listed first.
+
+It does not include time and location fields, because for scheduled trips those fields are already in [TripKey](#tripKey)
+
+### ScheduledCar
+
+Fields:
+
+- `run` (string, optional): The run number scheduled to the trip.
+- `operator` ([Operator](#operator), optional): The operator who is scheduled to operate the run that day, and therefore scheduled to do the trip.
```

Comment:
question: should this link to `.#operator` instead? From the above section, it looks like there's no other documentation for this field here.

```suggestion
- `operator` ([Operator](.#operator), optional): The operator who is scheduled to operate the run that day, and therefore scheduled to do the trip.
```